lib/Settings/Settings.h

```diff
@@ -21,7 +21,7 @@
 
 #define WEB_INDEX_FILENAME "/web/index.html"
 
-#define MILIGHT_GITHUB_USER "sidoh"
+#define MILIGHT_GITHUB_USER "nothingmn"
```

Comment:
I think the easiest is to just upload the file via `POST /web`. This is the command I use:

```
$ curl -vvv -X POST -F 'image=@data/web/index.html' http://milight-hub/web
```